src/Information.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import classNames from "classnames";
+
+function Information(props) {
+
+    const SummaryStyle = classNames ( "summary-item", {
+        "summary--orange": props.color === "orange",
+        "summary--yellow": props.color === "yellow",
+        "summary--green": props.color === "green",
+        "summary--blue": props.color === "blue",
+    })
+
+    const { summaryIcon, summaryText, summaryNote } = props;
+
```

Comment:
Vamos remover a linha 13 e usar o destructuing para receber esses valores via fucntion argument.

```suggestion
function Information({ summaryIcon, summaryText, summaryNote }) {

    const SummaryStyle = classNames ( "summary-item", {
        "summary--orange": props.color === "orange",
        "summary--yellow": props.color === "yellow",
        "summary--green": props.color === "green",
        "summary--blue": props.color === "blue",
    });
```


Aqui um doc da MDN sobre isso:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#binding_and_assignment